org.mwc.cmap.legacy/src/MWC/TacticalData/NarrativeEntry.java

```diff
@@ -317,16 +334,26 @@ public final String toString()
 //	}
 
 
+	protected void clearHash()
+	{
+	  _hashCode = null;
+	}
+	
 	@Override
   public int hashCode()
   {
-    final int prime = 31;
-    int result = 1;
-    result = prime * result + ((_DTG == null) ? 0 : _DTG.hashCode());
-    result = prime * result + ((_entry == null) ? 0 : _entry.hashCode());
-    result = prime * result + ((_track == null) ? 0 : _track.hashCode());
-    result = prime * result + ((_type == null) ? 0 : _type.hashCode());
-    return result;
+	  if(_hashCode == null)
+	  {
+	    final int prime = 31;
+	    int result = 1;
+	    result = prime * result + ((_DTG == null) ? 0 : _DTG.hashCode());

```

Comment:
Ooh, I have changed that.   I modified it so there was a single static intstance of NarrativeEntry._myInfo

That was a silly mistake, since _myInfo contains an instance of the object being edited.

Changing it back.